0006-sync-iteration/README.md

```diff
@@ -0,0 +1,197 @@
+0006: AT Protocol Sync v1.1
+============================
+
+*This is draft proposal, not the final specification. Some of the details, terminology, and behaviors might change, though we are pretty confident in the big picture.*
+
+One of the core concepts in atproto is Authenticated Transfer of public user content between services. This document summarizes an iteration on this part of the protocol, also called "Sync", which encompasses the repository, firehose, and CAR export mechanisms. The changes should improve and clarify behaviors, while being largely backwards-compatible and low-disruption to the existing network.
+
+The direct changes to core sync mechanisms are:
+
+- include additional metadata and MST blocks in firehose `#commit` messages to enable per-commit MST validation ("operation inversion" or "inductive firehose")
+- hard limits on repository diff (`#commit` message sizes), and removing the `tooBig` flag
+- new `#sync` firehose event type, for discontinuous changes in repo state
+- new `broken` account hosting status, for downstream services to indicate that repository synchronization is broken and needs repair
```

Comment:
I think this is just an old reference, now `desynchronized` and `throttled`.

Reply:
good catch! did a search and didn't see any other cases